src/model/NpmPackage.ts

```diff
@@ -1,12 +1,22 @@
 /* eslint-disable @typescript-eslint/no-type-alias */
-import { isArray, isBoolean, IsInterface, isString } from 'generic-type-guard';
-import type { GuardedType } from 'generic-type-guard';
+import {
+	type GuardedType,
+	isArray,
+	isBoolean,
+	IsInterface,
+	isString,
+	isUnion,
+	type TypeGuard,
+} from 'generic-type-guard';
```

Comment:
Linter started complaining here about a-z sorting, couldn't resolve it without merging the type imports into the module ones.